### ISSUE TITLE
Change license to ODBL

### ISSUE THREAD

**Issue by campoy** [src-d/datasets#65]
As per @eiso's comment  in https://github.com/src-d/guide/issues/204#issuecomment-386540245 we should change the current license (Apache v2) to ODBL.

**Comment by vmarkovtsev** [comment]
@campoy License of what exactly? Tools or index? Index is currently CC-BY-*

**Comment by smola** [comment]
Yes, tools need to preserve Apache 2. ODBL is only applicable to datasets (both index and siva files, but not the contents of siva file themselves such as commits and blobs, which preserve their own license and that needs to be noted too).

**Comment by vmarkovtsev** [comment]
The index file is Creative Commons Attribution-NonCommercialShareAlike 4.0 International as stated in the paper. Let's not change it until we present the paper on MSR.

**Comment by smola** [comment]
@vmarkovtsev AFAIK ODbl is less restrictive than Creative Commons Attribution-NonCommercialShareAlike 4.0 International, but if you want to be sure, just keep double licensing on the index.

**Comment by eiso** [comment]
Final to dual license but we should change the licenses **before releasing**

**Comment by smola** [comment]
Also keep in mind that you can add an additional license (as in double licensing) after releasing. But you cannot withdraw these licenses after releasing (or you can, but it probably has no legal effect).

**Comment by eiso** [comment]
Correct, do the dual license on the index file  (ODBL & Creative Commons Attribution-NonCommercialShareAlike 4.0 International), so that Vadim doesn't have to be concerned about the MSR paper, and future version ODBL only.

**Comment by campoy** [comment]
This was not done yet and it's pretty important.
If we agree the license is ODbL it should be mentioned somewhere in the README before we release this further.

**Comment by campoy** [comment]
We might have all agreed, but I don't see any reference to any licenses other than Apache v2.
Did I miss them?

**Comment by smola** [comment]
@campoy As far as I see, it was done only for Identifiers dataset, so this is just waiting for @campoy or @vmarkovtsev to do it ;)
